eduvpn/notify.py

```diff
@@ -11,10 +11,13 @@
 from repoze.lru import lru_cache
 from eduvpn.util import have_dbus
 from eduvpn.brand import get_brand
+from typing import Any, Optional
+from gi.repository import Notify
 
 
 @lru_cache(maxsize=1)
 def init_notify(lets_connect):
+	#type: (bool) -> Notify
```

Comment:
wrong indentation

Reply:
In some locations, the indentation seem to be unnecessarily changed.

Reply:
Mypy also couldn't detect this wrong indentation.
![image](https://user-images.githubusercontent.com/36157555/63028249-bf06ed80-beae-11e9-8ddd-a01e56ea4d3b.png)

Reply:
but what about pep8?